gbpservice/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -103,6 +102,8 @@ def post(self, **body):
             pecan.response.status = 400
             msg = ("Failed to serve HTTP post request %s %s."
                    % (self.method_name, str(err).capitalize()))
+            extra_import = ("need to remove this import %s" % config)
```

Comment:
config-agent module is restructured so I looked into other files. In main.py this import is there but it is using that and in other files which are making rpc call if they are not using this they are not importing it. I have checked removing this import  and maintaining this import in both case it sends same rpc call object to configurator. But need to be checked that configurator behaves same in both case or not.